yoti_python_sdk/config.py

```diff
@@ -30,3 +30,7 @@
 X_YOTI_SDK = "X-Yoti-SDK"
 X_YOTI_SDK_VERSION = X_YOTI_SDK + "-Version"
 JSON_CONTENT_TYPE = "application/json"
+PASSPORT = "PASSPORT"
```

Comment:
Could we change these to have something like `ANCHOR_VALUE_` in front of them, just so it's evident from just so they have a bit more context 👍 
```suggestion
ANCHOR_VALUE_PASSPORT = "PASSPORT"
```